### DISCUSSION TITLE
Bulk Selections Column Doesn't respect set{el}Attributes

### DISCUSSION THREAD

**Discussion by inmanturbo** [rappasoft/laravel-livewire-tables#786] [General]
Bulk selection column ignores `setTheadAttributes` and `setTdAttributes`.

Two easy options for fixing are to use conditional and override from values set with above methods, or add `setBulkSelectionsTdAttributes` and `setBulkSelectionsTheadAttributes`, possibly `setBulkSelectionsCheckboxAttributes`?

**Comment by dgillier** [comment]
Why not removing the "bg-gray-50" in the th class list of the bulk selection column ?

It will then inherit any background color class of thead ?

**Comment by santiagoavilez** [comment]
Have you find a solution for that issue? im having the same issue with the bulk actions columm...

**Reply by dgillier** [reply]
Sorry but no...
Need to change package code probably

**Comment by lrljoe** [comment]
Please see PR [1058](https://github.com/rappasoft/laravel-livewire-tables/pull/1058) which should be finished soon.
